src/megatron/bridge/training/checkpointing.py

```
@@ -1910,7 +1914,7 @@ def _load_checkpoint_from_path(
                     return 0, 0
             checkpoint_name = get_checkpoint_name(load_dir, iteration, release)
 
-        reader = FileSystemReader(checkpoint_name)
+        reader = _get_checkpoint_reader(checkpoint_name)
```

Flake8 (7.3.0)

[error] undefined name '_get_checkpoint_reader'

(F821)